test/test_macho.rb

```diff
@@ -451,14 +451,19 @@ def test_delete_rpath
       file = MachO::MachOFile.new(filename)
 
       refute_empty file.rpaths
-      orig_ncmds = file.ncmds
+      orig_ncmds = current_ncmds = file.ncmds
       orig_sizeofcmds = file.sizeofcmds
-      orig_npaths = file.rpaths.size
+      orig_npaths = current_npaths = file.rpaths.size
 
-      file.delete_rpath(file.rpaths.first)
-      assert_operator file.ncmds, :<, orig_ncmds
-      assert_operator file.sizeofcmds, :<, orig_sizeofcmds
-      assert_operator file.rpaths.size, :<, orig_npaths
+      file.rpaths.each do |rpath|
+        file.delete_rpath(rpath)
+        current_npaths -= 1
+        current_ncmds -= 1
+
+        assert_equal file.ncmds, current_ncmds
+        assert_equal file.rpaths.size, current_npaths
+        assert_operator file.sizeofcmds, :<, orig_sizeofcmds
```

Comment:
Is there an easy way to tell how much this should be changing with each deleted rpath? I could align this check more closely with the other two if there were.

Reply:
Hmm, I _think_ `rpath.cmdsize` should give you that.

Reply:
I was thinking of something that, but I was concerned that that's almost rigging the test? 😅

Also `rpath` in this block is a `String`, no?

Reply:
Ah, you're right....

Okay, let's just stick with `:<` for now. I'm sure there's a way to do this, but it's escaping me at the moment :slightly_smiling_face: